tests/unit/helpers/export.test.js

```diff
@@ -0,0 +1,310 @@
+
+    expect(exportArray).toEqual([
+      header,
+      ['Absence injustifiée', 'Horaire', '20/05/2019 08:00', '20/05/2019 10:00', 'Girafes - 75', '', 'Jean-Claude', 'VAN DAMME', ''],
+      ['Congé', 'Journalière', '20/05/2019', '20/05/2019', 'Etoiles - 75', '', 'Princess', 'CAROLYN', 'brbr'],
+    ]);
+  });
+});
+
+describe('exportBillsAndCreditNotesHistory', () => {
+  const header = ['Nature', 'Identifiant', 'Date', 'Id Bénéficiaire', 'Titre', 'Nom', 'Prénom', 'Id tiers payeur', 'Tiers payeur', 'Montant HT en €', 'Montant TTC en €', 'Services'];
+  const bills = [
+    {
+      number: 'FACT-0549236',
+      date: '2019-05-20T06:00:00.000+00:00',
+      customer: {
+        _id: ObjectID('5c35b5eb1a4fb00997363eb3'),
+        identity: {
+          title: 'Mme',
+          firstname: 'Mimi',
+          lastname: 'Mathy',
+        },
+      },
+      client: { _id: ObjectID('5c35b5eb7e0fb87297363eb2'), name: 'TF1' },
+      netInclTaxes: 389276.023,
+      subscriptions: [{
+        service: { name: 'Temps de qualité - autonomie' },
+        hours: 20,
+        exclTaxes: 389276.0208,
+        inclTaxes: 410686.201944,
+      }],
+    }, {
+      number: 'FACT-0419457',
+      date: '2019-05-22T06:00:00.000+00:00',
+      customer: {
+        _id: ObjectID('5c35b5eb1a6fb02397363eb1'),
+        identity: {
+          title: 'M',
+          firstname: 'Bojack',
+          lastname: 'Horseman',
+        },
+      },
+      client: { _id: ObjectID('5c35b5eb1a6fb87297363eb2'), name: 'The Sherif' },
+      netInclTaxes: 1057.1319439,
+      subscriptions: [{
+        service: { name: 'Forfait nuit' },
+        hours: 15,
+        exclTaxes: 700.0208,
+        inclTaxes: 738.521944,
+      }, {
+        service: { name: 'Forfait nuit' },
+        hours: 7,
+        inclTaxes: 302,
+        exclTaxes: 318.6099999,
+      }],
+    },
+  ];
+  const creditNotes = [
+    {
+      number: 'F1501231',
+      thirdPartyPayer: { _id: new ObjectID('5d761ad7ffd1dc0d39dadd7e'), name: 'SW' },
+      date: '2019-05-21T01:00:00.000+00:00',
+      customer: {
+        _id: new ObjectID('5d761a8f6f6cba0d259b17eb'),
+        identity: {
+          firstname: 'Jar jar',
+          lastname: 'Binks',
+        },
+      },
+      subscription: { service: { name: 'Temps de qualité - autonomie' } },
+      exclTaxesCustomer: 10.5,
+      inclTaxesCustomer: 5.5,
+      exclTaxesTpp: 8,
+      inclTaxesTpp: 3,
+    },
+    {
+      number: 'F6473250',
+      date: '2019-05-25T02:00:00.000+00:00',
+      customer: {
+        _id: new ObjectID('5d761a8f6f8eba0d259b173f'),
+        identity: {
+          lastname: 'R2D2',
+        },
+      },
+      subscription: { service: { name: 'Temps de qualité - autonomie' } },
+      exclTaxesCustomer: 10.5,
+      inclTaxesCustomer: 5.5,
+    },
+  ];
+  let mockBill;
+  let mockCreditNote;
+  let formatPriceStub;
+  let formatFloatForExportStub;
+
+  const expectBill = value => mockBill.expects('find')
```

Comment:
il vaut mieux mettre ca dans les `it` pour les `verify` apres

Reply:
fait